src/pint/pintk/pulsar.py

```diff
@@ -412,12 +415,32 @@ def add_jump(self, selected):
                 if self.fitted:
                     self.postfit_model.delete_jump_and_flags(None, num)
                 log.info("removed param", f"JUMP{str(num)}")
+                print("removed param", f"JUMP{str(num)}")
                 return toas_jumped
+
+            # Has to be some overlap between jumps and selected TOAs
+            elif np.any(toas_jumped & selected):
+                # if not, then they don't exactly match, delete the common subset
+                jumped_selected = toas_jumped & selected
+                # Post fit model and prefit model share the same TOA table, so as long as we
+                # don't delete the jump altogether, modifying prefit model table flags is fine.
+                self.prefit_model.delete_not_all_jump_toas(
+                    self.all_toas.table["flags"][jumped_selected], num
+                )
+                log.info(
+                    f"Removing existing jump JUMP{str(num)} from {jumped_selected.astype(int).sum()} TOAs"
+                )
+                print(
```

Comment:
here's another example

Reply:
Oh, I see, I misinterpreted this as having the option of both. I just left the `print` statement there to be a bit more informative. I can clean that up if required...

Reply:
Yeah, I think just the log.info's are enough.  Because the user is going to see the TOAs and/or JUMP disappear from the display as well.  So I think just zap the prints.

Reply:
ok, thanks

Reply:
@scottransom @dlakaplan I did change this to reflect jump changes as `log.info`. Can someone please approve the CI?